google/cloud/aiplatform/docker_utils/build.py

```diff
@@ -191,23 +185,18 @@ def _prepare_entrypoint(package: Package, python_command: str = "python") -> str
 
 
 def _prepare_package_entry(package: Package) -> str:
```

Comment:
Does this method still need to take package as a param?

Reply:
Removed.